build_library.sh

```diff
@@ -22,6 +22,8 @@ cmake_build $2
 if [ $1 == 'catkin' ]; then
     echo 'done. please run the following:'
     echo "  . $target/setup.bash"
+elif [ $1 == 'eigen' ]; then
+	cp -r $CMAKE_PREFIX_PATH/include/eigen3/* $CMAKE_PREFIX_PATH/include
```

Comment:
I compiled PCL and related ROS packages successfully even without that workaround. Why was it needed?

The [FindEIgen.cmake](https://github.com/PointCloudLibrary/pcl/blob/master/cmake/Modules/FindEigen.cmake#L14-L18) script shipped with PCL should be able to find the include directory inside `include/eigen3`.

Reply:
I don't remember right now exactly which package was complaining, but I think `pcl_ros` or some other ROS package that needs PCL was looking for Eigen's config file, and that's why I had to "install it" instead of just doing the copy, but there was another package that didn't find it in that location.

Reply:
I could successfully built the `catkin_ws` (with most pending PRs merged, except for `xacro`) and an overlay with those two lines commented out. I suggest to remove them.